weather_app/models.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import unicode_literals
+from django.db import models
+import uuid
```

Comment:
why an unused import, implement `pylint with pylint-django plugin` and `pycodestyle` for better and standard python code